CANMotorUnit.h

```diff
@@ -161,6 +161,17 @@ void AssembleMaxPIDPWMPacket(CANPacket *packetToAssemble,
     uint16_t PWMSetMax);
 uint16_t GetMaxPIDPWMFromPacket(CANPacket *packet);
 
+// Assembles a packet to set the angle of the PCA servo
+void AssemblePCAServoPacket(CANPacket *packetToAssemble,
+    uint8_t targetDeviceGroup,
+    uint8_t targetDeviceSerial,
+    uint8_t serverNum,
+    int32_t angle);
+// Get the angle value of the PCA server from the packet
+int32_t GetAngleValueFromPacket(const CANPacket *packet);
+// Get the servo number from the packet
+uint8_t GetServoNumFromPacket(const CANPacket *packet); 
```

Comment:
Can we changes these comments to doxy-style documentation? See `AssemblePotHiSetPacket()` for an example of what it should be like. Some of the other methods are older and don't have as great documentation, but new methods should have pretty thorough docs.